codeforamerica/vita-min
# MD 502: Add EIC to pdf and XML

## Link to pivotal/JIRA issue
- e.g. [link](https://codeforamerica.atlassian.net/browse/FYST-658?atlOrigin=eyJpIjoiZjQ5NGRhNTE4NTFlNGQ2MzhmNDVjOWZjMDA0Y2U4YzkiLCJwIjoiaiJ9)
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Added MD 502 EIC fields
## How to test?
- Select a DF persona that has qualifying children, laney_qss is one example. And if you want to test for someone that doesn't you can use zeus_two_w2s
- make them MFJ 
- see if EarnedIncomeCredit and MDEICWithQualChildInd are in the XML
- see in line 22 and 22b are in the pdf
## Screenshots (for visual changes)
- After
![Screenshot 2024-11-04 at 6 33 24 PM](https://github.com/user-attachments/assets/796899c5-dc99-40ba-8f55-648ef4374863)
![Screenshot 2024-11-04 at 6 33 33 PM](https://github.com/user-attachments/assets/bcce68d7-3ba1-4515-8494-e9d25cb5db3d)
